src/Features/CSharp/Portable/Completion/CSharpCompletionService.cs

```diff
@@ -34,32 +44,50 @@ internal class CSharpCompletionService : AbstractCompletionService
                 new CrefCompletionProvider(),
                 new SnippetCompletionProvider(),
                 new ExternAliasCompletionProvider(),
-            }.ToImmutableArray();
+                new OverrideCompletionProvider(),
+                new PartialCompletionProvider(),
+                new XmlDocCommentCompletionProvider()
+            );
+
+        private readonly Workspace _workspace;
 
-        public override IEnumerable<CompletionListProvider> GetDefaultCompletionProviders()
+        public CSharpCompletionService(Workspace workspace)
+            : base(workspace)
```

Comment:
If you pass workspace ot the base, why do you need to store it here?  Why not have hte base expose it through some protected member?